src/Be.Vlaanderen.Basisregisters.Shaperon/DbaseNullableInt32.cs

```diff
@@ -0,0 +1,75 @@
+namespace Be.Vlaanderen.Basisregisters.Shaperon
+{
+    using System;
+    using System.Globalization;
+    using System.IO;
+
+    public class DbaseNullableInt32 : DbaseFieldValue
+    {
+        public static readonly DbaseIntegerDigits MaximumIntegerDigits = DbaseInt32.MaximumIntegerDigits;
+
+        private int? _value;
+
+        public DbaseNullableInt32(DbaseField field, int? value = null) : base(field)
```

Comment:
The difference is, that with the non-nullables you make an explicit decision to either pass in a value or not (hence 2 ctors). The use case for not passing in a value is that you're going to set it later on, which is the majority of cases, since a dbase field value usually gets initialized as part of a record once, and is then set when using the record. If the record is reused (by reading binary data into it), then that makes the most sense. The overload that does take a non-nullable value is when you already know the value at construction time (granted, I've only had a need for that in very few cases).

The nullable variant can get away with one ctor since the `= null` or `= default` mechanism just works (and null is an expected value).